query-node/mappings/package.json

```diff
@@ -1,17 +1,21 @@
 {
   "name": "query-node-mappings",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "description": "Mappings for hydra-processor",
-  "main": "lib/mappings/index.js",
+  "main": "lib/mappings/src/index.js",
   "license": "MIT",
   "scripts": {
     "build": "rm -rf lib && tsc --build tsconfig.json",
     "lint": "echo \"Skippinng\"",
     "clean": "rm -rf lib"
   },
   "dependencies": {
-    "@dzlzv/hydra-common": "0.0.3",
-    "@polkadot/types": "~2.10.2-7",
+    "@dzlzv/hydra-common": "2.1.0-beta.7",
+    "@dzlzv/hydra-db-utils": "2.1.0-beta.7",
+    "@joystream/content-metadata-protobuf": "^1.0.0",
+    "@joystream/types": "^0.16.0",
+    "iso-639-1": "^2.1.8",
+    "query-node": "^0.1.0",
```

Comment:
the package by this name is the generated graphql-server `query-node/generated/graphql-server` and its version is 0.0.0
So what package is this supposed to be?

The build is failing because of this:
```
error Couldn't find package "query-node@^0.1.0" required by "query-node-mappings@0.1.0" on the "npm" registry.
```

Reply:
The problem is also that this package is not available until after the code is generated.